bundles/io/org.eclipse.smarthome.io.audio/src/main/java/org/eclipse/smarthome/io/audio/AudioFormat.java

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) 2015-2016 Harald Kuhn
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.audio;
+
+/**
+ * An audio format definition 
+ *
+ * @author Harald Kuhn (hkuhn42) initial api
+ */
+public class AudioFormat {
+
+    /**
+     * Codec
+     */
+    private String codec;
+
+    /**
+     * Container
+     */
+    private String container;
+
+    /**
+     * Big endian or little endian
+     */
+    private boolean bigEndian;
+
+    /**
+     * Bit depth or bit rate depending on codec
```

Comment:
Aren't there codecs (all?) that have a depth and a rate? What is expected in here in that case?
Is this a mandatory information? If not, I would use `Integer`, so that it can also be left `null`.